perceval/backends/core/askbot.py

```diff
@@ -471,6 +472,7 @@ def parse_answer_container(update_info):
             body = bs_answer.select("div.post-body")
             # Get the user information container and parse it
             update_info = body[0].select("div.post-update-info")
+            logging.info(f"User info: {update_info}")
```

Comment:
I think we can avoid this message because it will print the whole user information to the output.